src/main/java/smile/identity/core/ConfigHelpers.java

```diff
@@ -4,11 +4,11 @@
 public class ConfigHelpers {
     private final static String TEST_ENV = "0";
     private final static String PROD_ENV = "1";
-    private final static String TEST_SERVER = "https://testapi.smileidentity.com";
-    private final static String PROD_SERVER = "https://api.smileidentity.com";
+    private final static String TEST_SERVER = "https://testapi.smileidentity.com/v1/";
```

Comment:
If this lives here than the Java library will not be able to support verification

Reply:
i.e. `/v2/verify`

Reply:
good point, updated!